spec/system/purchasing_a_reservation_spec.rb

```diff
@@ -263,4 +263,25 @@
       )
     end
   end
+
+  describe "honours cutoff hours" do
+    let(:reserve_time) { 78.minutes.from_now }
+
+    before do
+      instrument.update!(cutoff_hours: 2)
+    end
+
+    it "cannot create a reservation before cutoff hours" do
```

Comment:
Should this read "... *after* cutoff hours?"

Reply:
yesss